DOC-1967: Fixed the mouse pointer style from a text cursor to a default arrow pointer when hovering over the tree dialog component items. (#2678)
tinymce/tinymce-docs: modules/ROOT/pages/6.4.2-release-notes.adoc

// {productname} 6.4.2 also includes the following bug fixes:

=== Fixed the mouse pointer style from a text cursor to a default arrow pointer when hovering over the tree dialog component items.

> In previous versions of {productname} while hovering over tree dialog component items, the mouse pointer style would render incorrectly.

Reviewer: ```suggestion
In previous versions of {productname}, while hovering over tree dialog component items, the mouse pointer style would render incorrectly.
```

> === Fixed the mouse pointer style from a text cursor to a default arrow pointer when hovering over the tree dialog component items.
> 
> In previous versions of {productname}, while hovering over tree dialog component items, the mouse pointer style would render incorrectly.
> 
> As a consequence, the mouse pointer would render as a text cursor rather than the default arrow pointer.
> 
> In {productname} 6.4.2, the styles for tree component items have been updated to correct this issue.

Reviewer 2: ```suggestion
=== When hovering over tree dialog components the moust pointer rendered incorrectly.

In previous versions of {productname}, when hovering over tree dialog component items, the mouse pointer style rendered incorrectly.

Specifically, the mouse pointer rendered as an I-beam pointer rather than the default arrow pointer.

In {productname} 6.4.2, the styles for tree component items have been updated to correct this issue.

NOTE: as of {productname} 6.4.2, tree components are only used as part of the xref:advanced-templates.adoc[Advanced Templates] Premium plugin.
```

Suggested edit.

> In {productname} 6.4.2, the styles for tree component items have been updated to corrected this issue.

Reviewer: ```suggestion
In {productname} 6.4.2, the styles for tree component items have been updated to correct this issue.
```

// {productname} 6.4.2 also includes the following bug fixes:

> === When hovering over tree dialog components the moust pointer rendered incorrectly.

Reviewer 2: ```suggestion
=== When hovering over tree dialog components the mouse pointer rendered incorrectly.
```

Typo correction.
